projects/packages/forms/src/blocks/contact-form/edit.js

```diff
@@ -140,6 +140,18 @@ export const JetpackContactFormEdit = forwardRef(
 			}
 		} );
 
+		useEffect( () => {
+			if (
+				! hasInnerBlocks &&
+				registerBlockVariation &&
+				! isPatternsModalOpen &&
+				window.location.search.indexOf( 'showPatterns' ) !== -1
+			) {
+				setIsPatternsModalOpen( true );
+			}
+			// eslint-disable-next-line react-hooks/exhaustive-deps
+		}, [] );
+
```

Comment:
Would we have a better way to do this?
The idea here is to be able to open the Patterns modal when the user enters the Editor.
That's something triggered by a button on the landing page.

Reply:
Hmm, it doesn't look too bad, I think it's fine. Just maybe let's call the var `showJetpackFormsPatterns` or something like that to minimize chances of any conflicts down the line.